neurom/core/tests/test_neuron.py

```diff
@@ -37,6 +37,9 @@
 SWC_PATH = Path(__file__).parent.parent.parent.parent / 'test_data/swc/'
 
 
+def test_simple():
+    nrn1 = nm.load_neuron(str(Path(SWC_PATH, 'simple.swc')))
```

Comment:
what does it test?

Reply:
That it does not crash... I can remove this test if you want.
I wanted to add this test before `test_load_neuron_pathlib` which is already a more specialized case.